mmdet/models/losses/accuracy.py

```diff
@@ -1,15 +1,17 @@
 import torch.nn as nn
 
 
-def accuracy(pred, target, topk=1):
+def accuracy(pred, target, topk=1, thresh=None):
     """Calculate accuracy according to the prediction and target
 
     Args:
-        pred (torch.Tensor): The model prediction.
-        target (torch.Tensor): The target of each prediction
+        pred (torch.Tensor): The model prediction, shape (N, num_class)
+        target (torch.Tensor): The target of each prediction, shape (N, )
         topk (int | tuple[int], optional): If the predictions in ``topk``
             matches the target, the predictions will be regarded as
             correct ones. Defaults to 1.
+        thresh (float, optional): If not None, threshold for pred with scores
+            under this value being incorrect. Default to None.
```

Comment:
This sentence is a little confusing. Is it better to say this?
`If not None, predictions with scores under this threshold are to be considered incorrect`